ebay-frontend/src/app/app.module.ts

```diff
@@ -1,12 +1,12 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
-
+import {MatToolbarModule} from '@angular/material/toolbar';
```

Comment:
Very minor, but we should put spaces around the imported item to be consistent with the other lines
`import { MatToolbarModule } from '@angular/material/toolbar`